cognite/neat/_rules/models/dms/_rules.py

```diff
@@ -421,15 +431,23 @@ def matching_version_and_space(cls, value: SheetList[DMSView], info: ValidationI
         if not (metadata := info.data.get("metadata")):
             return value
         model_version = metadata.version
-        if different_version := [view.view.as_id() for view in value if view.view.version != model_version]:
+        if different_version := [
+            view.view.as_id()
+            for view in value
+            if view.view.version != model_version and view.view.space not in COGNITE_SPACES
+        ]:
             for view_id in different_version:
                 warnings.warn(
                     PrincipleMatchingSpaceAndVersionWarning(
                         f"The view {view_id!r} has a different version than the data model version, {model_version}",
                     ),
                     stacklevel=2,
                 )
-        if different_space := [view.view.as_id() for view in value if view.view.space != metadata.space]:
+        if different_space := [
+            view.view.as_id()
+            for view in value
+            if view.view.space != metadata.space and view.view.space not in COGNITE_SPACES
```

Comment:
Not complain about version/space mismatch if the other view is CDM or IDM. User often has to include these core models to get a valid DML, so even though it is against our principle we need to skip it.